compiler/circlechef/tests/shape_signature/test.recipe

```diff
@@ -41,5 +41,15 @@ operation {
     activation: NONE
   }
 }
+operation {
```

Comment:
plz do not edit this file. this file is to check shape signature and has nothing to do with new options.
Op test recipe should be added into `res/CircleRecipes`

Reply:
> plz do not edit this file. this file is to check shape signature and has nothing to do with new options. Op test recipe should be added into `res/CircleRecipes`

Okay, I'll update it.